gp-nested-forms/gpnf-display-child-entries-table-format-all-child-fields.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Gravity Perks // GP Nested Forms // Display Table Format for All Fields
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * MODIFIED: Show *all* child fields.
+ *
+ * Plugin Name:  GP Nested Forms - Display Table Format for All Fields
+ * Plugin URI:   https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ * Description:  This snippet displays the Child entries of all Nested Forms field on a form in a table format.
+ * Author:       Gravity Wiz
+ * Version:      0.1
+ * Author URI:   https://gravitywiz.com
+ */
+add_action( 'init', function () {
+
+	if ( !is_callable( 'gp_nested_forms' ) ) {
+		return;
+	}
+
+	add_filter( 'gform_merge_tag_filter', function ( $value, $merge_tag, $modifiers, $field, $raw_value ) {
+
+		if ( $field->type != 'form' || $value === false ) {
+			return $value;
+		}
```

Comment:
Just realized this snippet doesn't support activating this via the `gpnf_table` modifier. Can we add support for that? I believe we just need to replace these highlighted lines with these: 

https://github.com/gravitywiz/snippet-library/pull/61/files#diff-54e2bcf2704f530e900c8e5f416f3468733ff75e0d32c5455cb6575ad0f5f91cR22-R24

Reply:
I've updated the snippet with the highlighted lines, however, this version of the snippet actually works without the gpnf_table modifier. Using the modifier and updating the snippet with the lines you suggested, still displays the child entries in the same table format as when no modifier is used. 

Also, there already is a check for !is_callable( 'gp_nested_forms' ) on line 17, which is also present in the line I just added. I think we may have to delete one, but I wanted to confirm if the highlighted lines are still needed since the modifier doesn't affect the snippet.